src/ch/imedias/rsccfx/view/RsccHomePresenter.java

```diff
@@ -39,6 +39,8 @@ public void initSize(Scene scene) {
     view.supportViewBtn.prefHeightProperty().bind(scene.heightProperty());
     view.requestViewBtn.prefWidthProperty().bind(scene.widthProperty().divide(2));
     view.requestViewBtn.prefHeightProperty().bind(scene.heightProperty());
+    view.requestImgView.fitWidthProperty().bind(scene.widthProperty().divide(4));
```

Comment:
This was already in a code fragment... since we are going to redesign the start page anyway, shall I still fix it?